src/util/RequestsUtil.ts

```diff
@@ -93,4 +95,32 @@ export class RequestsUtil {
 		}
 		return this.hashCode( resolver.tag ) & 0x00FFFFFF;
 	}
+
+	public static async checkTicketValidity( ticketKeyString: string ): Promise<boolean> {
+		try {
+			const searchResults = await MojiraBot.jira.issueSearch.searchForIssuesUsingJqlGet( {
+				jql: `(${ BotConfig.request.invalidRequestJql }) AND key in (${ ticketKeyString })`,
+				fields: ['key'],
+			} );
+			const invalidTickets = searchResults.issues.map( ( { key } ) => key );
+			if ( invalidTickets.length > 0 ) {
+				return false;
+			} else {
+				return true;
+			}
+		} catch ( err ) {
+			this.logger.error( err );
+			return false;
+		}
+	}
+
+	public static checkTicketLinks( content: string ): number {
```

Comment:
I wonder if we actually need this function at all, we already have `RequestEventHandler#getTickets` which does essentially the same thing already but better (though arguably more confusing).